src/os/bluestore/BlueFS.cc

```diff
@@ -1967,6 +2013,7 @@ void BlueFS::_compact_log_async(std::unique_lock<ceph::mutex>& l)
   t.op_jump(log_seq, new_log_jump_to);
 
   // allocate
+  //FIXME: check if we want DB here?
```

Comment:
I guess WAL is never slower then DB, but it would be more confusing.

Reply:
Another uncompleted TODO. IMO originally it should be log_file->fnode.prefer_bdev here instead of BDEV_DB. As log file to be located primarily at WAL device.